wdyr.js

```diff
@@ -0,0 +1,20 @@
+// Implements Why Did You Render (WDYR) in Dev
+
+import React from 'react';
+import Config from 'react-native-config';
+import lodashGet from 'lodash/get';
+
+const useWDYR = lodashGet(Config, 'USE_WDYR', 'false') === 'true';
+
+if (useWDYR && process.env.NODE_ENV === 'development') {
+    const whyDidYouRender = require('@welldone-software/why-did-you-render');
+    whyDidYouRender(React, {
+        // Include and exclude components to be tracked by their displayName here
+        include: [
+            /^Avatar/,
+            /^ReportActionItem/,
+            /^ReportActionItemSingle/,
```

Comment:
There is A LOT of noise when tracking every component and I'm not sure that opening my console to 100+ notifications about unrelated components would be that useful. 

I think that tracking relevant components we are working on might be a better default.

Reply:
I understand, but also the feature is opt-in. So maybe the default would be to show everything unless otherwise specified. It's not difficult to filter for the exact component by filtering in the JS console. Mostly bringing this up because I have used this tool before and don't usually futz with the configs too much.

Reply:
Maybe as a compromise we can just mention how to show everything in a commen? :)

Reply:
The original GH did specify:
> Implement WDYR and only enable it on certain "problem components" like ReportActionItem, Avatar etc.

But I agree having a comment here about how to enable it for everything would be nice to have.

Reply:
Well, what can I say, my past and future selves don't always get along ! 😂

Reply:
Fair enough! I made changes to display everything by default and left the comment on how to include/exclude certain components only.